feat(changelog): functions-fixed-missing-logs-in-console 2023-01-19 (#1220)
scaleway/docs-content: changelog/january2023/2023-01-19-functions-fixed-missing-logs-in-console.mdx

---
title: Missing logs in console and API
status: fixed
author:

>     fullname: 'undefined'

Reviewer: ```suggestion
    fullname: 'Join the #serverless-functions channel on Slack.'
```

>     url: 'undefined'

Reviewer: ```suggestion
    url: 'https://slack.scaleway.com'
```

>     url: 'undefined'

Reviewer: ```suggestion
    url: 'Join the #serverless-functions channel on Slack.'
```

date: 2023-01-19
category: serverless
product: functions
---

> Some users experienced an issue on the console and API where logs where not appearing. This bug is now fixed.

Reviewer 2: ```suggestion
Users were experiencing an issue on the console and the API where logs were not appearing. This bug is now fixed.
```

> We also recommend to use Serverless Functions dashboards in [Scaleway Cokpit](https://www.scaleway.com/en/docs/observability/cockpit/quickstart/) to monitor all your metrics and logs and access advanced features (ordering, filtering ...)

Reviewer 2: ```suggestion
We also recommend you use the Serverless Functions dashboards in the [Scaleway Cockpit](https://www.scaleway.com/en/docs/observability/cockpit/quickstart/) to monitor all your metrics and logs and access advanced features (ordering, filtering...)
```
